benchto-service-docker/config/graphite/storage-schemas.conf

```diff
@@ -0,0 +1,10 @@
+# Schema definitions for whisper files. Entries are scanned in order,
+# and first match wins.
+#
+[carbon]
+pattern = ^carbon\.
+retentions = 1s:1d,1min:90d
+
+[default]
+pattern = .*
+retentions = 1s:1d,1min:6d,10min:1800d
```

Comment:
This is not optimal configuration... `1s` granularity is what matters. `10min` is useless. I'd propose

```
1s:5d,1min:6d
```

the value from keeping "1min" is only that it servers as example